plutus-core/plutus-ir/test/PlutusIR/Transform/StrictLetRec/Tests.hs

```diff
@@ -20,17 +20,15 @@ import PlutusIR.Transform.StrictLetRec.Tests.Lib (defaultCompilationCtx,
                                                   pirTermFromFile)
 import System.FilePath.Posix (joinPath, (</>))
 import Test.Tasty (TestTree)
-import Test.Tasty.Extras (runTestNestedIn, testNested)
+import Test.Tasty.Extras (embed, runTestNested, testNested)
 import Test.Tasty.HUnit (testCase, (@?=))
 import UntypedPlutusCore.Evaluation.Machine.Cek (EvaluationResult (..))
 
 path :: [FilePath]
 path = ["plutus-ir", "test", "PlutusIR", "Transform"]
 
 test_letRec :: TestTree
-test_letRec = runTestNestedIn path do
-  testNested
-    "StrictLetRec"
+test_letRec = runTestNested path . pure $ testNested "StrictLetRec"
```

Comment:
Could've been `runTestNested (path ++ ["StrictLetRec"])`, but I'm again using the `pure` pattern.